src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+
+html {
```

Redundant empty line at the begging of the file
```suggestion
html {
```